### DISCUSSION TITLE
Can TestNextMajor download dependency apps from appsource?

### DISCUSSION THREAD

**Discussion by HaSvensson** [microsoft/AL-Go#1072] [Q&A]
When performing TestNextMajor I would like to have e.g. LS Central downloaded from appsource instead of using parameter InstallApps or appDependencyProbingPaths. In that way I will also be able to perform TestNextMajor against my other dependeny apps, not only BC.
Is that possible?

**Comment by freddydk** [comment]
AL-Go for GitHub automatically downloads dependencies in form of symbols from AppSource.
This means that you can build your apps, but symbols cannot be published to a container and as such, you won't be able to run tests in a container without having access to a runtime package from the partner.
